packages/react-router/src/tests/utilities.ts

```diff
@@ -1,52 +0,0 @@
-import {RouterChildContext} from 'react-router-dom';
```

Comment:
I could not find these being used, https://grokt.shopify.io/results?q=createDefaultHistory%28%29, and the API has significantly changed. I'm not sure how to port them.

I decided to remove them (if we can)

Reply:
The package exposes a single entrypoint, and the stuff in this folder is not referenced from it. It's good to burn it all